app/models/user.rb

```diff
@@ -10,7 +10,7 @@ class User < ActiveRecord::Base
   validates :first_name, presence: true
   validates :last_name, presence: true
 
-  def record_nominations(nominations_hash)
+  def record_nominations(user_id, nominations_hash)
```

Comment:
This is redundant. You're in the User model, so just call `self`. That represents whatever User object is having the method run on it.